account_chatter/views/account_account_views.xml

```diff
@@ -16,5 +17,13 @@
             </xpath>
         </field>
     </record>
-
+    <record id="view_account_form_inherit_hidden" model="ir.ui.view">
```

Comment:
same ID as inherited view, the view with the group should be the one with a different ID